assets/js/components/Dashboard/PaMessaging/NewPaMessagePage/NewPaMessagePage.tsx

```diff
@@ -43,6 +54,15 @@ const NewPaMessagePage = () => {
     setAudioPlaying(true);
     fetchAudioPreview(phoneticText.length ? phoneticText : visualText).then(
       async (audioData) => {
+        if (audioData.status !== 200) {
```

Comment:
`response` might be a better name for this variable, as it's not just the raw data.